### ISSUE TITLE
Records from import to be deleted

### ISSUE THREAD

**Issue by kitenetter** [BiologicalRecordsCentre/iRecord#482]
Request to delete a set of records that were imported incorrectly.

Please delete 101 records:
- iRecord user 16801, 
- records added to survey iRecord | Anthomyiidae Study Group
- record IDs within range 8880020 to 8880120 (all from a single import as far as I'm aware)

@johnvanbreda  : this is another operation that I'd be happy to take on myself with some guidance, but I would also need to have permissions to delete records from the database, which obviously carries some risk. If you think that is worth pursuing let me know and we can look at this when we skype.

**Comment by kitenetter** [comment]
I don't think we managed to do this, which is perhaps just as well since I entered an incorrect user ID above. It should be user id 84079. I am awaiting confirmation that these record do still need deleting (I think they do) and will advise when I've heard from them.

There are 101 records defined by:

`SELECT *
FROM occurrences o
WHERE o.id > 8880019 AND o.id < 8880121 AND o.created_by_id = 84079`
